Bottom: 2e345b3deafe60d622e4e997e493b83d96be2bc8
Top:    5cd566c631548baf63fbb58e1de537352861ce12
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-27 21:49:38 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
